### ISSUE TITLE
image jpg/png

### ISSUE THREAD

**Issue by zhao-lun** [sunaku/simple-ftp#2]
Hi sunaku, Thank you for this ftp code. Is image supported? I tried the code, the transfer was success but the data receive was corrupted.

**Comment by sunaku** [comment]
No, I had removed support for binary data transfers in commit d6ab17959eb92b7cb65e836ec9bfd6c3eac75e97.  You can try reverting that commit.
